.github/workflows/main.yaml

```diff
@@ -391,4 +392,61 @@ jobs:
           name: ${{ matrix.container-runtime }}-e2e-test-logs
           path: |
             /tmp/cloudcore.log
-            /tmp/edgecore.log
\ No newline at end of file
+            /tmp/edgecore.log
+
+  conformance_e2e_test:
+      runs-on: ubuntu-22.04
+      timeout-minutes: 180
+      name: conformance e2e test
+      needs: image-prepare
+      env:
+        GO111MODULE: on
+      steps:
+        - name: Install Go
+          uses: actions/setup-go@v3
+          with:
+            go-version: 1.20.x
+
+        - uses: actions/cache@v3
+          with:
+            path: ~/go/pkg/mod
+            key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+        - name: Checkout code
+          uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
+
+        - name: for test fetch upstream
```

Comment:
What is the purpose of this step?

Reply:
This is only used for debugging in the forked repository, it's not necessary in the final PR.
If the review passes, I will remove this and the debug-triggering branch :) 
@Shelley-BaoYue

Reply:
done